{
  "uid" : "10d1900485098ef6",
  "name" : "确认密码输入为空",
  "fullName" : "testcase.test_login.TestLogin#test_login01",
  "historyId" : "1ee099853c08e83b466b85648f89aead",
  "time" : {
    "start" : 1650509990569,
    "stop" : 1650509990569,
    "duration" : 0
  },
  "status" : "failed",
  "statusMessage" : "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntestcase\\test_login.py:81: AssumptionFailure\n>>\tpytest.assume(flag == False)\nAssertionError: assert False",
  "statusTrace" : "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               testcase\\test_login.py:81: AssumptionFailure\nE               >>\tpytest.assume(flag == False)\nE               AssertionError: assert False\n\nD:\\Python\\lib\\site-packages\\six.py:718: FailedAssumption",
  "flaky" : false,
  "newFailed" : false,
  "newBroken" : false,
  "newPassed" : false,
  "retriesCount" : 0,
  "retriesStatusChange" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntestcase\\test_login.py:81: AssumptionFailure\n>>\tpytest.assume(flag == False)\nAssertionError: assert False",
    "statusTrace" : "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               testcase\\test_login.py:81: AssumptionFailure\nE               >>\tpytest.assume(flag == False)\nE               AssertionError: assert False\n\nD:\\Python\\lib\\site-packages\\six.py:718: FailedAssumption",
    "steps" : [ {
      "name" : "预期与结果",
      "time" : {
        "start" : 1650509990569,
        "stop" : 1650509990569,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "f5ea1d7d356499c4",
        "name" : "预期",
        "source" : "f5ea1d7d356499c4.json",
        "type" : "application/json",
        "size" : 49
      }, {
        "uid" : "c277cf0031b505bb",
        "name" : "结果",
        "source" : "c277cf0031b505bb.json",
        "type" : "application/json",
        "size" : 49
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 2,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : true
    }, {
      "name" : "响应失败原因",
      "time" : {
        "start" : 1650509990569,
        "stop" : 1650509990569,
        "duration" : 0
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "8d4cc170f36d7f",
        "name" : "数据",
        "source" : "8d4cc170f36d7f.txt",
        "type" : "text/plain",
        "size" : 65
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : true
    }, {
      "name" : "断言结果",
      "time" : {
        "start" : 1650509990569,
        "stop" : 1650509990584,
        "duration" : 15
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : false
    } ],
    "attachments" : [ {
      "uid" : "2f4b264a909e2744",
      "name" : "接口路径",
      "source" : "2f4b264a909e2744.attach",
      "type" : "text/plain",
      "size" : 42
    }, {
      "uid" : "ba919ae8b0ac51af",
      "name" : "请求参数",
      "source" : "ba919ae8b0ac51af.attach",
      "type" : "text/plain",
      "size" : 160
    }, {
      "uid" : "1707610e9eed4fe",
      "name" : "预期结果",
      "source" : "1707610e9eed4fe.attach",
      "type" : "text/plain",
      "size" : 49
    }, {
      "uid" : "17879815933311c2",
      "name" : "响应状态",
      "source" : "17879815933311c2.attach",
      "type" : "text/plain",
      "size" : 16
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 7,
    "shouldDisplayMessage" : true,
    "stepsCount" : 3,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "注册接口"
  }, {
    "name" : "feature",
    "value" : "master"
  }, {
    "name" : "parentSuite",
    "value" : "testcase"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "Mikey"
  }, {
    "name" : "thread",
    "value" : "13688-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "testcase.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "case",
    "value" : "{'case_id': 4, 'case_title': '确认密码输入为空', 'url': 'http://127.0.0.1:6666/business/regist_form', 'case_data': '{\"username\":\"admin01\",\"password\":\"admin\",\"re_password\":\"\",\"phone\":\"15736884464\",\"sex\":\"女\",\"birthday\":\"1995-04-03\",\"qq\":\"12345@qq.com\",\"email\":\"19999@163,com\"}', 'res': '{\"code\": 1005, \"msg\": \"重复密码不能为空\"}', 'app_name': '注册接口', 'demo': '啊啊'}"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "10d1900485098ef6.json",
  "parameterValues" : [ "{'case_id': 4, 'case_title': '确认密码输入为空', 'url': 'http://127.0.0.1:6666/business/regist_form', 'case_data': '{\"username\":\"admin01\",\"password\":\"admin\",\"re_password\":\"\",\"phone\":\"15736884464\",\"sex\":\"女\",\"birthday\":\"1995-04-03\",\"qq\":\"12345@qq.com\",\"email\":\"19999@163,com\"}', 'res': '{\"code\": 1005, \"msg\": \"重复密码不能为空\"}', 'app_name': '注册接口', 'demo': '啊啊'}" ]
}